julia/bin/plot_bb_beams.jl

```diff
@@ -0,0 +1,138 @@
+using CUDASIMDTypes
+using CairoMakie
+using HDF5
+using SixelTerm
+
+dir = "/tmp"
+prefix = "blue_"
+iter = "0000000"
```

Comment:
It's fairly clear to me what this is doing and why these variables are set as they are.   But I could see this being confusing to someone who isn't also looking at the config files generating the input to this script.   Maybe just reference the config that's generating this data?

Reply:
Done!